mongomock/collection.py

```diff
@@ -1576,6 +1631,10 @@ def _extend_collection(out_collection, field, expression):
                                         if field not in doc_dict:
                                             doc_dict[field] = []
                                         doc_dict[field].extend(values)
+                                    elif operator == '$pushAll':
```

Comment:
It looks like the same implementation than the $push operator above: I think one of them is wrong. Let's fix this.

Reply:
After checking, pushAll is not a valid group operator.
https://docs.mongodb.com/manual/reference/operator/aggregation/group/#pipe._S_group